src/main/java/no/nav/pto/veilarbportefolje/cv/CvService.java

```diff
@@ -0,0 +1,59 @@
+package no.nav.pto.veilarbportefolje.cv;
+
+import io.micrometer.core.instrument.Counter;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.arbeid.cv.avro.Cv;
+import no.nav.pto.veilarbportefolje.database.BrukerRepository;
+import no.nav.pto.veilarbportefolje.domene.AktoerId;
+import no.nav.pto.veilarbportefolje.elastic.ElasticIndexer;
+import no.nav.pto.veilarbportefolje.kafka.KafkaConsumerService;
+import no.nav.pto.veilarbportefolje.registrering.RegistreringService;
+import org.joda.time.DateTime;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+import static no.nav.metrics.MetricsFactory.getMeterRegistry;
+
+@Slf4j
+public class CvService implements KafkaConsumerService<Cv> {
+
+    private final RegistreringService registreringService;
+    private final BrukerRepository brukerRepository;
+    private final ElasticIndexer elasticIndexer;
+    private final Counter antallBrukereSomIkkeHarDeltCv;
+    private final Counter antallBrukereSomHarDeltCv;
+
+    public CvService(RegistreringService registreringService, BrukerRepository brukerRepository, ElasticIndexer elasticIndexer) {
+        this.registreringService = registreringService;
+        this.brukerRepository = brukerRepository;
+        this.elasticIndexer = elasticIndexer;
+        antallBrukereSomIkkeHarDeltCv = Counter.builder("portefolje_har_ikke_delt_cv").register(getMeterRegistry());
+        antallBrukereSomHarDeltCv = Counter.builder("portefolje_har_delt_cv").register(getMeterRegistry());
+    }
+
+    @Override
+    public void behandleKafkaMelding(Cv cv) {
+        AktoerId aktoerId = AktoerId.of(cv.getAktoerId());
+        Optional<ZonedDateTime> registreringOpprettet = registreringService.hentRegistreringOpprettet(aktoerId);
+        if (!harDeltCvMedNav(cv.getSistEndret(), registreringOpprettet)) {
```

Comment:
vad händer om en brukare går ut och in från oppfolgingen men inte gör ändringar i cv ?

Reply:
De må dele cv på nytt, men vet ikke om de havner på denne topicen så burde antagelig  sette cv delt til 'N'

Reply:
0a9f744465e0b4fefe1c9d2dce0390f9b50b7c29